src/lib/strings/index.ts

```diff
@@ -20,3 +13,40 @@ export function hash(str: string): number {
         return newHash & newHash;
     }, 0);
 }
+
+/**
+ * This util will replace the spaces in the last 5/9ths of the string with
```

Comment:
Moved this comment from the test, since it's more helpful here, and more consistent